Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CPUPercenageGauge.cs

```diff
@@ -48,7 +48,18 @@ public double GetValueAndReset()
                 var baseValue = this.lastCollectedTime.Ticks - previouslyCollectedTime.Ticks;
                 baseValue = baseValue != 0 ? baseValue : 1;
 
-                value = (float)((this.lastCollectedValue - previouslyCollectedValue) / baseValue * 100.0);
+                var diff = this.lastCollectedValue - previouslyCollectedValue;
+
+                if (diff < 0)
```

Comment:
We have rate counters https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/RateCounterGauge.cs#L100 It will not be logical to get negative number of requests/sec:

https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/75373e57dcf8d646c54ee188461c373f2cc98939/Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CounterFactory.cs#L31-L35